streaming/pom.xml

```diff
@@ -95,7 +95,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.18</version>
-        <!-- Note config is repeated in scalatest config -->
+        <!-- Note config is repeated in scala test config -->
```

Comment:
Please add license header in this POM file

Reply:
added